sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 ## 12.0.2 (Unreleased)
 
-- Share a single httpClient among clients. [PR #6657](https://github.com/Azure/azure-sdk-for-js/pull/6657)
-  Currently clients share their parent client's pipeline. But when the pipeline doesn't have a http client specified, a `DefaultHttpClient` will be created for each of the clients. This leads to memory leaks and exhaustion of TCP connections when many clients are created with `keepAlive` option enabled because each client creates a long-live http connection.  This change creates a default http client instance to share among all clients if none is specified in the pipeline options.
+- Service clients now share a single http client instance by default. [PR #6657](https://github.com/Azure/azure-sdk-for-js/pull/6657)
```

Comment:
continue discussion in https://github.com/Azure/azure-sdk-for-js/pull/6721#discussion_r362286453

would "Service clients" cause confusion with BlobServiceClient?

Reply:
Possible, but I think adding service to client helps disambiguate this usage of client vs an http client. Feel free to take or leave the suggestions though!